src/main/java/net/worldseed/multipart/model_bones/misc/ModelBoneHitbox.java

```diff
@@ -62,7 +62,7 @@ public void updateOldViewer(@NotNull Player player) {
                     }
                 };
 
-                this.stand.setTag(WSEE, "hitbox");
```

Comment:
I'd prefer to not change this tag value, and this shouldn't be needed since bones have a `getName` method